src/partials/setupInstructions.html

```diff
@@ -4,23 +4,88 @@
             <h2 class="title is-2 light-text is-spaced pb-40">Setup instructions</h2>
         </div>
     </div>
-    <div class="box">
-        <h3 class="subtitle is-5">Load balancer</h3>
-        <ul>
-            <li>Download latest release: <a target="_blank" href="https://github.com/NodeFactoryIo/vedran-daemon/releases" >github.com/NodeFactoryIo/vedran/releases</a></li>
-            <li>Create polkadot account - <a target="_blank" href="https://wiki.polkadot.network/docs/en/learn-account-generation">wiki.polkadot.network/docs/en/learn-account-generation</a></li>
-            <li>./vedran start --auth-secret=&#60;generate random secret&#62; --private-key=&#60;private seed from generated account&#62;&#60;/private&#62;</li>
-            <li>Use --help for additional configuration and flags</li>
-        </ul>
+    <nav class="tabs is-centered is-medium is-boxed">
+        <div class="container">
+            <ul>
+                <li class="tab is-active" id="windowsTab"><a>Windows</a></li>
+                <li class="tab" id="linuxTab"><a>Linux</a></li>
+                <li class="tab" id="macTab"><a>Mac</a></li>
+            </ul>
+        </div>
+    </nav>
+    <div class="container section">
+        <div id="windowsContainer" class="content-tab is-active">
+            <div class="box">
+                <h1>Windows</h1>
+                <h3 class="subtitle is-5">Load balancer</h3>
+                <ul>
+                    <li>Download latest release: <a target="_blank" href="https://github.com/NodeFactoryIo/vedran-daemon/releases" >github.com/NodeFactoryIo/vedran/releases</a></li>
```

Comment:
```suggestion
                    <li>Download latest release from <a target="_blank" href="https://github.com/NodeFactoryIo/vedran-daemon/releases" >Github</a></li>
```

Reply:
<img width="999" alt="Screenshot 2021-02-05 at 13 02 50" src="https://user-images.githubusercontent.com/51483483/107031471-83eb2f00-67b2-11eb-9d54-9d27f1889175.png">

Reply:
Seems like part of my comment was missing. I meant to say, if @MakMuftic or @mpetrun5  Could add you terminal commands for linux/mac to download and unzip with wget and zip tool or something (maybe even command to install unzip)